docs/nodes/AI_ML/CLASSIFICATION/ONE_HOT_ENCODING/ONE_HOT_ENCODING.md

```diff
@@ -39,19 +37,15 @@ import outputImg from './examples/EX1/output.jpeg'
 <Example1 />
 
 <SectionBreak />
-  
-    
 
 [//]: # (Appendix)
 
+## Appendix
```

Comment:
This render to the correct location? The reason that I ask is that it's really just moving relative to the `import` directives and those will presumably disappear (only the stuff that is imported should remain). Let's just confirm because this might be a common change and something that you could do immediately across _all_ files and just get it sorted.